.github/workflows/workspace.yml

```diff
@@ -0,0 +1,81 @@
+name: Workspace
```

Comment:
For this file, should we be setting `RUSTFLAGS: "-Dwarnings"`? Clippy issues are warnings, and we are pinning the version, so it seems like we should deny warnings.

Reply:
You are right. But I added it only for the Clippy job, since it's not needed for the other two.